### PR TITLE
Add namespaced deployment* labels

### PR DESCRIPTION
**Pull request by kargakis**
Ref issue: https://github.com/openshift/origin/issues/3605

@ironcladlou @deads2k @smarterclayton

### CODE REVIEW

File: pkg/deploy/util/util.go [hunk 1]
```diff
@@ -141,14 +141,18 @@ func MakeDeployment(config *deployapi.DeploymentConfig, codec runtime.Codec) (*a
 		selector[k] = v
 	}
 	selector[deployapi.DeploymentConfigLabel] = config.Name
+	selector[deployapi.DeprecatedDeploymentConfigLabel] = config.Name
 	selector[deployapi.DeploymentLabel] = deploymentName
+	selector[deployapi.DeprecatedDeploymentLabel] = deploymentName
```
Comment (liggitt) [inline pkg/deploy/util/util.go]: This won't select old things that only had the deprecated labels, will it?

Comment (liggitt) [inline pkg/deploy/util/util.go]: Can we even do this now if we don't have OR capability in selectors?

Comment (kargakis) [inline pkg/deploy/util/util.go]: Darn, forgot that all labels from the selector need to exist in labels... Right.

Comment (kargakis) [inline pkg/deploy/util/util.go]: Yeah, that was rushed, OR selectors need to be implemented upstream?.

**Pull request closed without merging**